check/nbformat

```diff
@@ -25,26 +25,34 @@ for arg in $@; do
 done
 
 # Get the working directory to the repo root.
-cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-cd "$(git rev-parse --show-toplevel)"
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" || exit
+cd "$(git rev-parse --show-toplevel)" || exit
 
 # Install a pinned version of tf-docs
 TF_DOCS_COMMIT=b3dc8a922d8bdc6e998a8ad4f4953359dc6e576a
-TF_DOCS_INSTALLED=$(pip list | grep  -c $TF_DOCS_COMMIT)
+TF_DOCS_INSTALLED="$(pip list | grep -c "$TF_DOCS_COMMIT")"
+BLACK_INSTALLED="$(pip list | grep -c 'black-nb')"
 
 if [ "$TF_DOCS_INSTALLED" != "1" ]; then
   pip install -U git+https://github.com/tensorflow/docs@$TF_DOCS_COMMIT
 fi
 
+if [ "$BLACK_INSTALLED" != "1" ]; then
+    pip install -U black-nb
+fi
+
 FORMAT_CMD="python3 -m tensorflow_docs.tools.nbfmt --indent=1"
+BLACK_FORMAT="black-nb"
```

Comment:
You're introducing a new format command, but you are not taking into account consistent variable naming - e.g. the other one ends with CMD this one ends with FORMAT. Also, this one is scoped to the tool (BLACK) while the original is left generic "FORMAT_CMD" (which made sense before, as it was the only one). It probably should be renamed by the tool as well - e.g. `TFDOCS_FORMAT`.